ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/fullyprivate/proxy/OWNERS

```diff
@@ -0,0 +1,12 @@
+approvers:
+- jianlinliu
+- Amoghrd
```

Comment:
still add Amoghrd ?

Reply:
I just copied the OWNER file from the existing folders to here, if you are suggesting to remove him, we can have dedicated PR to do that.

Reply:
+1
small and simple PRs are easier to review / track